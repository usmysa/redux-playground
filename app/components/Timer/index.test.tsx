import { renderWithProviders } from "@/__tests__";
import { Timer } from ".";

describe("Timer", () => {
  it("renders correctly", () => {
    const { container } = renderWithProviders(<Timer width={100} />);

    expect(container).toMatchSnapshot();
  });
});
