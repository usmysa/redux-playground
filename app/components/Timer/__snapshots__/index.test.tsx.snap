// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Timer > renders correctly 1`] = `
<div>
  <progress
    class="styles_timer__7z3f421"
    max="30"
    style="--styleVars_width__7z3f420: 100px;"
    value="30"
  />
</div>
`;
